src/main.rs

```diff
@@ -65,14 +86,14 @@ fn main() {
         match data.get_mut::<UserMap>() {
             Some(user_map) => {
                 let mut markov = user_map.entry(author.id.0).or_insert(Markov::new());
```

Comment:
Use the functional form of `Markov::new`. Using this method, you create a new `Markov` every time this is called, regardless of whether or not it is actually used.